test_datasets/10x_SmartSeq2_pbmc_GSE132044/path_data.csv

```diff
@@ -0,0 +1,3 @@
+name,type,path,stats
```

Comment:
Can you create this file in the `download.R` file so that paths are not hardcoded?